### ISSUE TITLE
Write it in shell?

### ISSUE THREAD

**Issue by Jack47** [Augusto111/STtech_test#4]
After writing it in shell, you may think about the pros and cons of these two ways

**Comment by Augusto111** [comment]
I'm not very familiar with shell, I'll try it.

**Comment by Augusto111** [comment]
I have already written a shell script, this method is easier than the python script, after all, the shell script is native command in Linux.

**Comment by Jack47** [comment]
Cool, well done. Could you please try to find other shell ways to do this? 
For example, use commands like `find`, to avoid  `ls $1`.

**Comment by Augusto111** [comment]
yeah, I use "find [target directory path] -name '*'" to implement this feature.
But why should we avoid to use the method like "ls", for code readability or effectiveness？

**Comment by Augusto111** [comment]
yeah, I use "find [target directory path] -name '*'" to implement this feature.

**Comment by Jack47** [comment]
How about `find target-dir -executable -exec bash -c $0 {} \;` ?

**Comment by Augusto111** [comment]
Well, we can used "-perm -111" to match a subset of permissions. Like:
find target-dir -type f -perm -111 -exec $0 {} \;
find target-dir -type f -perm -111 -exec bash -c ‘$0’ {} \;
find target-dir -type f -perm -111 -exec bash -c "\$0" {} \;
Maybe this one line command can do all of things……
